pkgs/development/interpreters/cling/default.nix

```diff
@@ -34,36 +34,37 @@
 let
   stdenv = clangStdenv;
 
-  # The LLVM 9 headers have a couple bugs we need to patch
-  fixedLlvmDev = runCommand "llvm-dev-${llvmPackages_9.llvm.version}" { buildInputs = [git]; } ''
-    mkdir $out
-    cp -r ${llvmPackages_9.llvm.dev}/include $out
-    cd $out
-    chmod -R u+w include
-    git apply ${./fix-llvm-include.patch}
-  '';
+  version = "0.9";
```

Comment:
This should probably be `1.0` to pick up the latest support. We may need to ping the Cling team to get them to cut a Git release with a tag.